settings.gradle

```diff
@@ -0,0 +1,28 @@
+
+pluginManagement {
+    repositories {
+        maven {
+            // RetroFuturaGradle
+            name "GTNH Maven"
+            url "http://jenkins.usrv.eu:8081/nexus/content/groups/public/"
```

Comment:
I'm still not a fan of this, but I'll let it pass.

Reply:
You know who to yell at.  😉